pages/dkp/kommander/2.2/release-notes/index.md

```diff
@@ -2,10 +2,185 @@
 layout: layout.pug
 navigationTitle: Kommander 2.2 Release Notes
 title: Kommander 2.2 Release Notes
-menuWeight: 0
+menuWeight: 10
 excerpt: View release-specific information for Kommander 2.2
 enterprise: false
 beta: false
 ---
 
-This section includes the Release Notes for Kommander 2.2
+**D2iQ&reg; Kommander&reg; (DKP&reg;) version 2.2 was released on March 16, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+[Download](../../download/) and [install](../../install/) the latest version to get started.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download Kommander. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install this product.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in DKP.
+
+Kommander 2.2 supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster you want to attach using Kommander 2.1 must be running a Kubernetes version in that range. 
+
+### New features and capabilities
+
+#### DKP major version upgrade
+
+Version 2.2 represents a major upgrade that moves forward D2IQ's architecture to give you access to the next generation of centralized Kubernetes and smart cloud native applications. It incorporates ClusterAPI as a major re-architecture in its management of production Kubernetes clusters. [ClusterAPI](https://cluster-api.sigs.k8s.io/introduction.html), or CAPI, enables declarative creation, configuration, and management of clusters. Declarative mode is a Kubernetes best practice that simplifies lifecycle tasks, making them more consistent and more repeatable. 2.2 enhances your existing clusters to use a new architecture.
+
+For more information on planning your upgrade, see the [DKP Major Version upgrade](/dkp/konvoy/2.2/major-version-upgrade/) guide.
+
+#### Provision Kubernetes Infrastructure from DKP
+
+This version allows for provisioning and managing of Kubernetes clusters, making it easier than ever to get your infrastructure up and running quickly.
+
+For more information on provisioning, see [Managing Clusters](../../clusters/).
+
+#### Continuous deployment
```

Comment:
Let's remove every section in this heading. All of these features were for 2.1. So what we need is just one prompt that says #### Feature 1 to remind people where to put info later.